### PR TITLE
Part2

### PR DESCRIPTION
**Pull request by morecallan**
@alexsanchez728 - it looks like this is ready for review. Can you confirm?

### CODE REVIEW

**Comment by alexsanchez728** [comment]
@morecallan Yep, ready to review!
https://my-angular-contacts-fc3cb.firebaseapp.com/#!/login

File: db/contacts.json [hunk 1]
```diff
@@ -0,0 +1,94 @@
+{
+  "contacts": [
+    {
```
Comment (morecallan) [inline db/contacts.json]: This should be an object of objects with keys for each contact (just like we will be getting back from Firebase in the real world).

File: db/contacts.json [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+      "lastName": "Sanchez",
+      "preferedName": "Flex Sanchez",
+      "email": "alesanchez728@gmail.com",
+      "primaryPhone": "(708)-307-1983",
+      "address": "1500A Norvel Ave. Nashville, TN",
+      "birthday": "10-03-1995"
+    },
+    {
+      "firstName": "Dummy",
+      "lastName": "McDumbface",
```
Comment (morecallan) [inline db/contacts.json]: Make these guys unique (at least somewhat) so you can make sure that stuff is working later. 

ALSOM YOU'RE A DUMMY MCDUMBFACE

File: javascripts/controllers/LoginCtrl.js [hunk 3]
```diff
@@ -1,6 +1,14 @@
...
 
-app.controller("LoginCtrl", function ($scope) {
-  $scope.controller = "LoginCtrl";
-  console.log("hello, LoginCtrl");
+app.controller("LoginCtrl", function ($location, $rootScope, $scope, AuthService) {
+  $scope.authenticate = () => {
+    AuthService.authenticateGoogle().then((result) => {
+      $rootScope.uid = result.user.uid;
+      $scope.$apply(() => {
+        $location.url("/search");
```
Comment (morecallan) [inline javascripts/controllers/LoginCtrl.js]: Oops you don't have a `search` route in this project!

**Review by morecallan — COMMENTED** [review]
@alexsanchez728 - Nice! A few small changes to push up then I'll merge it for ya!

File: javascripts/controllers/LoginCtrl.js [hunk 4]
```diff
@@ -5,7 +5,7 @@ app.controller("LoginCtrl", function ($location, $rootScope, $scope, AuthService
     AuthService.authenticateGoogle().then((result) => {
       $rootScope.uid = result.user.uid;
       $scope.$apply(() => {
-        $location.url("/search");
+        $location.url("/view");
```
Comment (morecallan) [inline javascripts/controllers/LoginCtrl.js]: Oops. Still not don't have `/view` ... you have `/contacts/view` though!

**Review by morecallan — COMMENTED** [review]
@alexsanchez728 - Left one more small comment. Push that up and I'll merge it!

**Comment by morecallan** [comment]
👍 beautiful!

**Pull request merged**